controllers/clusteroperator.go

```diff
@@ -58,6 +58,20 @@ func relatedObjects() []osconfigv1.ObjectReference {
 	}
 }
 
+// operandVersions returns the current list of OperandVersions for the
+// ClusterOperator objects's status.
+func (r *ProvisioningReconciler) operandVersions() []osconfigv1.OperandVersion {
+	if r.ReleaseVersion == "" {
```

Comment:
Why can't we define an empty OperandVersion before we check for the env var. If the env var is defined we add values to the previously declared OperandVersion. That way, we only have to return once. Something like this perhaps?

```
        operandVersion := []osconfigv1.OperandVersion{}
        if r.releaseVersion != "" {
                operandVersions = append(operandVersions, osconfigv1.OperandVersion{Name: "operator", Version: r.releaseVersion})
        } 
        return operandVersion
```

Reply:
sure, i can do that